subsys/bluetooth/mesh/Kconfig

```diff
@@ -192,6 +192,38 @@ config BT_MESH_UNPROV_BEACON_INT
 
 if BT_MESH_PB_ADV
 
+config BT_MESH_PB_ADV_USE_RELAY_SETS
+	bool "Use relay advertising sets to send provisioning PDUs"
+	depends on BT_MESH_RELAY_ADV_SETS > 0
+	help
+	  Use relay advertising sets to send provisioning PDUs
+
+config BT_MESH_PB_ADV_TRANS_PDU_RETRANSMIT_COUNT
+	int "Link Open and Transaction PDU retransmit count"
+	default 7 if BT_MESH_PB_ADV_USE_RELAY_SETS
+	default 0
+	range 0 7
+	help
+	  Controls the number of retransmissions of original Link Open and Transaction PDU,
+	  in addition to the first transmission.
+
+config BT_MESH_PB_ADV_TRANS_ACK_RETRANSMIT_COUNT
+	int "Link Ack and Transaction Ack retransmit count"
+	default 0
+	range 0 7
+	help
+	  Controls the number of retransmissions of original Link Ack and Transaction Acknowledgment PDU,
+	  in addition to the first transmission.
+
+config BT_MESH_PB_ADV_LINK_CLOSE_RETRANSMIT_COUNT
+	int "Link Close retransmit count"
+	default 7 if BT_MESH_PB_ADV_USE_RELAY_SETS
```

Comment:
Why are so many retransmission attempts required in the case of relay set usage? both link close and pdu sending? 
Especially pdu sending. Growing the number of attempts from 0 to 7 seems strange. Any reason for this?

Reply:
Will terminate if ack received.

Reply:
Good point. 0 to 7 seems strange for provision, due to Provision not define such as `xmit`, but for current implementation, just use mesh message `xmit` also for provision pdu sending, could change this strange method to more general ways, but not in this PR, thanks.